articles/100_ament.md

```diff
@@ -85,10 +167,21 @@ Therefore various features of catkin have been designed to be similar to rosbuil
 
 catkin has many advantages over rosbuild.
 Those cover out-of-source builds, automatic CMake config file generation, an installation target and many more.
-But for some cases it is not flexible enough or not as easy to use as it should be.
+
+Over time, however, feedback about the shortcomings of catkin have been collected.
```

Comment:
Should it be *feedback [..] has been collected*? Afaik feedback is always singular.